### PR TITLE
Kr lnvgy 

### PR DESCRIPTION
**Pull request by bwgartner**
A new RC document with another IHV partner - Lenovo
- has two bundled commits (one IHV-centric and one for the general components to pull in this content)
- after your review/approval/modifications, please
  - publish in docserv for a quick review
  - then push to main doc site

With this said, it is still using the old stylesheets like all the other kubernetes/reference RI/RC docs and then the next big branch/PR will be to update all of these towards the new TRD stylesheets

### CODE REVIEW

File: kubernetes/reference/DC-kubernetes_rc_k3s-slemicro_lnvgy-se350-se450 [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+MAIN="SA.adoc"
+# Stylesheet Root
+STYLEROOT="/usr/share/xml/docbook/stylesheet/suse2021-ns"
+# Format Type
+ADOC_TYPE="book"
+# Turn on postprocessing
+ADOC_POST="yes"
+# Leverage SUSE Best Practices
+XSLTPARAM="--stringparam publishing.series=sbp"
+DOCBOOK5_RNG_URI="http://docbook.org/xml/5.1/rng/docbookxi.rnc"
```
Comment (tomschr) [inline kubernetes/reference/DC-kubernetes_rc_k3s-slemicro_lnvgy-se350-se450:25]: I would suggest to use DocBook 5.2:

```suggestion
DOCBOOK5_RNG_URI="http://docbook.org/xml/5.2/rng/docbookxi.rnc"
```

File: kubernetes/reference/DC-kubernetes_rc_k3s-slemicro_lnvgy-se350-se450 [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+## -------------------------------
+
+##
+# Basics
+##
+
+# Main Document 
+MAIN="SA.adoc"
+# Stylesheet Root
+STYLEROOT="/usr/share/xml/docbook/stylesheet/suse2021-ns"
```
Comment (tomschr) [inline kubernetes/reference/DC-kubernetes_rc_k3s-slemicro_lnvgy-se350-se450:18]: If you want to use the new stylesheets (not sure if this is applicable for this guide), use this:

```suggestion
STYLEROOT="/usr/share/xml/docbook/stylesheet/trd"
```

File: kubernetes/reference/DC-kubernetes_rc_k3s-sles_lnvgy-se350-se450 [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+MAIN="SA.adoc"
+# Stylesheet Root
+STYLEROOT="/usr/share/xml/docbook/stylesheet/suse2021-ns"
+# Format Type
+ADOC_TYPE="book"
+# Turn on postprocessing
+ADOC_POST="yes"
+# Leverage SUSE Best Practices
+XSLTPARAM="--stringparam publishing.series=sbp"
+DOCBOOK5_RNG_URI="http://docbook.org/xml/5.1/rng/docbookxi.rnc"
```
Comment (tomschr) [inline kubernetes/reference/DC-kubernetes_rc_k3s-sles_lnvgy-se350-se450:25]:
```suggestion
DOCBOOK5_RNG_URI="http://docbook.org/xml/5.2/rng/docbookxi.rnc"
```

File: kubernetes/reference/DC-kubernetes_rc_k3s-sles_lnvgy-se350-se450 [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+## -------------------------------
+
+##
+# Basics
+##
+
+# Main Document 
+MAIN="SA.adoc"
+# Stylesheet Root
+STYLEROOT="/usr/share/xml/docbook/stylesheet/suse2021-ns"
```
Comment (tomschr) [inline kubernetes/reference/DC-kubernetes_rc_k3s-sles_lnvgy-se350-se450:18]: Same here:

```suggestion
STYLEROOT="/usr/share/xml/docbook/stylesheet/trd"
```

**Review by tomschr — COMMENTED** [review]
Hi @bwgartner, looks fine, just found some minor issues. I think, using DocBook 5.2 to validate would be better. For the other part (TRD stylesheets) it depends a bit on whether you want to do it now or later.

**Comment by bwgartner** [comment]
> Hi @bwgartner, looks fine, just found some minor issues. I think, using DocBook 5.2 to validate would be better. For the other part (TRD stylesheets) it depends a bit on whether you want to do it now or later.

Was hoping to just get this published as is (matching all the other RI/RCs for now). I will take all of your suggested changes to do the massive set of RI/RCs next.

**Comment by tomschr** [comment]
> Was hoping to just get this published as is (matching all the other RI/RCs for now). I will take all of your suggested changes to do the massive set of RI/RCs next.

It was just a suggestion. :slightly_smiling_face: If you want to do it all at once, that's all fine. I just wanted to make sure all is prepared for the next update.

File: kubernetes/reference/adoc/i/SA.adoc [hunk 5]
```diff
@@ -47,7 +47,7 @@ ifdef::IHV-Fujitsu[link:{vn_Fujitsu_homePage}[{vn_Fujitsu}] (R)]
 ifdef::IHV-HPE[link:{vn_HPE_homePage}[{vn_HPE}] (R)]
 ifdef::IHV-HPQ[link:{vn_HPQ_homePage}[{vn_HPQ}] (R)]
 ifdef::IHV-IBM[link:{vn_IBM_homePage}[{vn_IBM}] (R)]
-ifdef::IHV-Lenovo[link:{vn_Lenovo_homePage}[{vn_Lenovo}] (R)]
+ifdef::IHV-LNVGY[link:{vn_LNVGY_homePage}[{vn_LNVGY}] (R)]
 ifdef::IHV-SMCi[link:{vn_SMCi_homePage}[{vn_SMCi}] (R)]
 as the platform for physical, baremetal, hypervisors and virtual machines
```
Comment (dariavladykina) [inline kubernetes/reference/adoc/i/SA.adoc:52]:
```suggestion
as the platform for physical, bare metal, hypervisors and virtual machines
```

File: kubernetes/reference/adoc/IHV/LNVGY/SA-RA-Deployment-Hardware.adoc [hunk 6]
```diff
@@ -0,0 +1,35 @@
+
+The base, starting configuration can reside all within a single {vn_LNVGY} Edge Server. Based upon the relatively small resource requirements for a
+ifdef::focusRancher[{pn_Rancher}]
+ifdef::focusK3s[{pn_K3s}]
+ifdef::focusRKE1[{pn_RKE1}]
+ifdef::focusRKE2[{pn_RKE2}]
+deployment, a viable approach is to deploy as a virtual machine (VM) on the target nodes, on top of an existing hypervisor, like KVM. For a physical host, there are tools that can be used during the setup of the server to provision virtual machines, or similar steps can be used to deploy the software stack on the baremetal system.
```
Comment (dariavladykina) [inline kubernetes/reference/adoc/IHV/LNVGY/SA-RA-Deployment-Hardware.adoc:7]:
```suggestion
deployment, a viable approach is to deploy as a virtual machine (VM) on the target nodes, on top of an existing hypervisor, like KVM. For a physical host, there are tools that can be used during the setup of the server to provision virtual machines, or similar steps can be used to deploy the software stack on the bare-metal system.
```

**Review by dariavladykina — COMMENTED** [review]
Hi Bryan, please see some suggestions (really minor things). Thank you!

File: kubernetes/reference/adoc/IHV/LNVGY/SE450.adoc [hunk 7]
```diff
@@ -0,0 +1,18 @@
+
+
+==== {vn_LNVGY} {familyLNVGY-Edge} {modelLNVGY-SE450}
+
+NOTE: The  {familyLNVGY-Edge} {modelLNVGY-SE450} is {companyName} YES Certified 
+Hardware.
+
+link:{modelLNVGY-SE450URL}[{vn_LNVGY} {familyLNVGY-Edge} {modelLNVGY-SE450}]::
+The {familyLNVGY-Edge} {modelLNVGY-SE450} is a single-socket server, with a 2U height and short depth case that can go almost anywhere. It can be mounted on a wall, placed on the floor like a tower server, or mounted in a rack. This rugged edge server can handle continuous operating temperatures from 5°C and 45°C; and some configurations are designed to meet NEBS Level-3 and ETSI requirements for 96 hours operating excursions from -5°C and 55°C as well as tolerance to locations with high-dust and vibration. The {familyLNVGY-Edge} {modelLNVGY-SE450} is based on the 3rd Gen Intel Xeon Scalable processor and is designed to virtualize traditional IT applications as well as new transformative AI systems, providing the processing power, storage, accelerator, and networking technologies required for today’s edge workloads.
```
[the remark this answers is not in the document]
Comment (bwgartner) [reply kubernetes/reference/adoc/IHV/LNVGY/SE450.adoc:9]: good fixes

File: kubernetes/reference/adoc/IHV/LNVGY/SE450.adoc [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+The {familyLNVGY-Edge} {modelLNVGY-SE450} is a single-socket server, with a 2U height and short depth case that can go almost anywhere. It can be mounted on a wall, placed on the floor like a tower server, or mounted in a rack. This rugged edge server can handle continuous operating temperatures from 5°C and 45°C; and some configurations are designed to meet NEBS Level-3 and ETSI requirements for 96 hours operating excursions from -5°C and 55°C as well as tolerance to locations with high-dust and vibration. The {familyLNVGY-Edge} {modelLNVGY-SE450} is based on the 3rd Gen Intel Xeon Scalable processor and is designed to virtualize traditional IT applications as well as new transformative AI systems, providing the processing power, storage, accelerator, and networking technologies required for today’s edge workloads. 
+
+** 10-36 server cores with up to 1TB of memory
+** broad-wired and wireless connectivity
+** supports up to four single-width GPUs or two double-wide GPUs
+** offers up to four PCIe 4.0 slots plus a slot dedicated to the OCP adapter
+** offer additional physical security features like chassis intrusion switch and lockable front beze
```
[the remark this answers is not in the document]
Comment (bwgartner) [reply kubernetes/reference/adoc/IHV/LNVGY/SE450.adoc:15]: more nice updates/fixes

**Review by dariavladykina — APPROVED** [review]

**Review by tomschr — APPROVED** [review]
Looks fine. :+1:

**Pull request merged**